atuin-client/src/import/resh.rs

```diff
@@ -0,0 +1,56 @@
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+pub struct ReshEntry {
+    pub cmd_line: String,
+    pub exit_code: i64,
+    pub shell: String,
+    pub uname: String,
+    pub session_id: String,
+    pub home: String,
+    pub lang: String,
+    pub lc_all: String,
+    pub login: String,
+    pub pwd: String,
+    pub pwd_after: String,
+    pub shell_env: String,
+    pub term: String,
+    pub real_pwd: String,
+    pub real_pwd_after: String,
+    pub pid: i64,
+    pub session_pid: i64,
+    pub host: String,
+    pub hosttype: String,
+    pub ostype: String,
+    pub machtype: String,
+    pub shlvl: i64,
+    pub timezone_before: String,
+    pub timezone_after: String,
+    pub realtime_before: f64,
+    pub realtime_after: f64,
+    pub realtime_before_local: f64,
+    pub realtime_after_local: f64,
+    pub realtime_duration: f64,
+    pub realtime_since_session_start: f64,
+    pub realtime_since_boot: f64,
+    pub git_dir: String,
+    pub git_real_dir: String,
+    pub git_origin_remote: String,
+    pub git_dir_after: String,
+    pub git_real_dir_after: String,
+    pub git_origin_remote_after: String,
+    pub machine_id: String,
+    pub os_release_id: String,
+    pub os_release_version_id: String,
+    pub os_release_id_like: String,
+    pub os_release_name: String,
+    pub os_release_pretty_name: String,
+    pub resh_uuid: String,
+    pub resh_version: String,
+    pub resh_revision: String,
+    pub parts_merged: bool,
+    pub recalled: bool,
+    pub recall_last_cmd_line: String,
+    pub cols: String,
+    pub lines: String,
+}
```

Comment:
I'd love to be able to get some of this stored in Atuin too :thinking: Obviously not required for this PR, but definitely a future thing